esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -2574,6 +2576,10 @@ bool CWsWorkunitsEx::onWUFile(IEspContext &context,IEspWULogFileRequest &req, IE
                     winfo.getWorkunitXml(req.getPlainText(), mb);
                     if (opt < 2)
                     {
+                        unsigned long xmlSize = mb.length();
+                        if (xmlSize > MAX_WU_XML_SIZE_FOR_BROWSE)
+                            throw MakeStringException(ECLWATCH_CANNOT_OPEN_FILE, "The xml (%ld bytes) is too big for viewing. Please download.", xmlSize);
```

Comment:
This wording seems to assume this is called from browser, not SOAP.  But not sure what the wording should be.  Is it really compressed vs uncompressed.  Rather than download vs browse?